lib/bn.js

```diff
@@ -459,6 +465,11 @@ BN.prototype.toJSON = function toJSON() {
   return this.toString(16);
 };
 
+BN.prototype.toBuffer = function toBuffer(endian, length) {
+  assert(typeof BN._Buffer !== 'undefined');
```

Comment:
Is there any point in exposing `_Buffer` to users? What do you think about making at a context variable instead?